x-pack/test/functional/services/ml/common_ui.ts

```diff
@@ -285,6 +285,20 @@ export function MachineLearningCommonUIProvider({
       await this.assertRowsNumberPerPage(testSubj, rowsNumber);
     },
 
+    async getEuiDescriptionListDescriptionFromTitle(testSubj: string, title: string) {
+      const subj = await testSubjects.find(testSubj);
+      const titles = await subj.findAllByTagName('dt');
+      const descriptions = await subj.findAllByTagName('dd');
+
+      for (let i = 0; i < titles.length; i++) {
+        const titleText = (await titles[i].parseDomContent()).html();
+        if (titleText === title) {
+          return (await descriptions[i].parseDomContent()).html();
+        }
+      }
+      return null;
+    },
+    
```

Comment:
In this blank line there is an indent which causes eslint to fail:
```suggestion

```

Reply:
Fixed in https://github.com/elastic/kibana/pull/109274/commits/936a18d22787a1d5e2916e3e1c45519ca8834f99